docs/command/mongocli-cloud-manager-automation-watch.txt

```diff
@@ -14,6 +14,11 @@ mongocli cloud-manager automation watch
 
 Watch for automation changes to be completed.
 
+This command checks the automation goal version periodically until it's reached.
+Once the expected status is reached, the command prints "Changes deployed successfully."
```

Comment:
Which changes? Not sure if this is the place to amend this message, but it seems vague. Changes to what?

Reply:
I'm not sure we want to change the output of the command here. It seems to me that changes deployed to make sense since we're checking versions diff in the automation?